contributors/devel/development.md

```diff
@@ -279,20 +279,38 @@ To build binaries for all platforms:
 make cross
 ```
 
+#### Verification
+```
+# Run all hack/verify-*.sh
+make verify
+```
+
 #### Test
 
 ```sh
 cd $working_dir/kubernetes
 
+# Run all hack/update-*.sh
```

Comment:
Do you mean `make update`? This is a comment for `make update`, which runs all hack/update-*.sh scripts.